pioneer/packages/app-tokenomics/src/lib/getValidatorData.ts

```diff
@@ -0,0 +1,51 @@
+import { ApiPromise } from '@polkadot/api';
+import { Vec } from '@polkadot/types';
+import { AccountId, IndividualExposure } from '@polkadot/types/interfaces';
+
+const getValidators = async (api: ApiPromise): Promise<any> => {
```

Comment:
The return value could be perhaps derived by TypeScript compiler itself if you omitted the `: Promise<any>`